content/md/first-steps/login.md

```diff
@@ -17,6 +17,17 @@ Check the option `Remember me on this computer` to memorise your credentials.
 
 ![image](../img/Login1.png)
 
+#  Login using Single Sign-On _(EE only)_
+
+Since the 3.0, your PIM administrator can activate the **Single Sign-On** feature (aka SSO). This feature will allow you to connect to your PIM using your corporate credentials.
+Once this feature is activated, you will no longer be prompted for your login and password in the PIM.
+
+To learn more about Single Sign-On, it's [here](configure-saml-sso.html)!
```

Comment:
not sure that the link "(configure-saml-sso.html)" will work

Reply:
If there is an article with this id, should be working fine.
Since we reworked the help center to work on two branches (v2 and v3), ways to build links have changed. ;)

Reply:
This is the article that was writen in this PR https://github.com/akeneo/pim-helpcenter/pull/285